itest/lnd_coop_close_external_delivery_test.go

```diff
@@ -10,13 +10,32 @@ import (
 )
 
 func testCoopCloseWithExternalDelivery(ht *lntest.HarnessTest) {
-	ht.Run("set delivery address at open", func(t *testing.T) {
+	ht.Run("set P2WPKH delivery address at open", func(t *testing.T) {
 		tt := ht.Subtest(t)
-		testCoopCloseWithExternalDeliveryImpl(tt, true)
+		testCoopCloseWithExternalDeliveryImpl(
+			tt, true, lnrpc.AddressType_UNUSED_WITNESS_PUBKEY_HASH,
+		)
 	})
-	ht.Run("set delivery address at close", func(t *testing.T) {
+
+	ht.Run("set P2WPKH delivery address at close", func(t *testing.T) {
```

Comment:
Nice tests - can we add `success := ht.Run` and stop the test if `!success`? this is helpful for future development as it's easier to debug that way.

Reply:
Thanks! Done:

```go
        ok := ht.Run("set P2WPKH delivery address at open", func(t *testing.T) {
                tt := ht.Subtest(t)
                testCoopCloseWithExternalDeliveryImpl(
                        tt, true, lnrpc.AddressType_UNUSED_WITNESS_PUBKEY_HASH,
                )
        })
        // Abort the test if failed.
        if !ok {
                return
        }

...
```